doc/3/controllers/document/create/snippets/create.java

```diff
@@ -0,0 +1,9 @@
+
+  ConcurrentHashMap<String, Object> document = new ConcurrentHashMap<>();
+  document.put("firstname", "John");
+  document.put("lastname", "Smith");
+
+  ConcurrentHashMap<String, Object> options = new ConcurrentHashMap<>();
+
+  kuzzle.getDocumentController().create("nyc-open-data", "yellow-taxi", document, "some-id", options)
+  .get();
```

Comment:
Keep the snippet simple: remove the unused `options` objects instantiation and parameter.